web_scraper/extract.py

```diff
@@ -38,11 +38,11 @@ def load_all_data(conn: extensions.connection) -> set:
             raise ValueError("No urls were found!")
         url_list = [url[0] for url in urls]
         return convert_to_set(url_list)
-    
+
 
 def convert_to_set(urls: list[str]) -> set:
     """Turns the list into a set to remove duplicate entries."""
-    
+
     return set(urls)
```

Comment:
Does this need to be it's own function?

Reply:
We added this as a separate function for testing